### DISCUSSION TITLE
Old zigbee-herdsman version 0.35.1 used, 0.39.0 available

### DISCUSSION THREAD

**Discussion by thomaszub** [Koenkk/zigbee2mqtt#21890] [General]
Hi,

I noticed that zigbee-herdsman version 0.35.1 is used while the newest available version is 0.39.0. Is there any particular
reason for not updating?

I have currently an issue with a timeout with the Deconz adapter that possibly could be solved with an update of zigbee-herdsman 0.39.0 (at least there are some fixes in the DeconzAdapter class):
https://github.com/Koenkk/zigbee2mqtt/issues/19576

**Comment by Koenkk** [comment]
It's available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html), and will be included in the 1 April z2m release.

**Comment by thomaszub** [comment]
Awesome news. Thank you!
